bodhi/server/scripts/untag_branched.py

```diff
@@ -53,15 +49,13 @@ def main(argv=sys.argv):
     log = logging.getLogger(__name__)
 
     settings = get_appsettings(config_uri)
-    engine = engine_from_config(settings, 'sqlalchemy.')
-    Session = scoped_session(sessionmaker(extension=ZopeTransactionExtension()))
-    Session.configure(bind=engine)
+    initialize_db(settings)
     db = Session()
     koji = buildsys.get_session()
     one_day = timedelta(days=1)
     now = datetime.utcnow()
 
-    with transaction.manager:
+    try:
```

Comment:
I'd prefer to keep it as-is since I don't want to just re-raise the exception. It is a little bit repetitive, but I'm not sure it's worth adding more complexity and abstraction for this sort of thing.